### DISCUSSION TITLE
自定义顶栏动态组件 tab 优化

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1598] [功能建议 / Ideas]
这四个 tab 能不能和之前提过的 BiliPlus 按钮(#593) 一样做成可右键单击可中键点击的。个人特别喜欢用中键，也不能少一步不用切 tab 再点击
![image](https://user-images.githubusercontent.com/49396421/108956699-19d7f280-76ab-11eb-9c18-a34bc6f8698c.png)

**Comment by the1812** [comment]
主要是左键对不上
'可右键单击可中键点击的'其实就是因为那是个链接, 但是这里左键点击是切换 tab 而不是打开链接

**Reply by Geeyun-JY3** [reply]
那就让当前 tab 可中键和右键点击？

**Reply by the1812** [reply]
当前 tab 可以, 中键是后台打开标签页吗?

**Reply by Geeyun-JY3** [reply]
就和 #593 的 BiliPlus 按钮一样。对，中键行为默认是当前窗口后台打开标签页。

**Reply by the1812** [reply]
ok
